src/randomizer.py

```diff
@@ -415,7 +415,9 @@ def random_type(random: Random) -> SimpleType:
     new_names: List[str] = []
     if random_bool(random, 0.5):
         new_names.append("unsigned")
-    new_names.append(random.choice(["char", "short", "int", "int"]))
+    new_names.extend(
+        random.choice(["char", "short", "int", "int", "long", "long long"]).split()
```

Comment:
hm, maybe this gives a bit too much weight to long and long long?
how about
```suggestion
        random_weighted(random, [
            (["char"], 1),
            (["short"], 1),
            (["int"], 2),
            (["long"], 0.5),
            (["long", "long"], 0.5),
        ])
```